accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+
+Here are some examples of SDK workloads and the packs that they might include:
+
+| Workload ID | Content|
+|--|--|
+| xamarin-android | Framework, runtime, build logic, tools and templates for Xamarin.Android.|
+| xamarin-forms | Templates for Xamarin.Forms and the Xamarin.Forms NuGet package used by the templates.<br><br>This would also include any packs necessary to build and run Xamarin.Forms projects. That could be done by depending on the xamarin-ios, xamarin-android and uwp components, or perhaps instead some kind of lightweight Xamarin.Forms host. |
+| xamarin-forms-android | Templates for Xamarin.Forms and the Xamarin.Forms NuGet package used by the templates.<br><br>This would also include any packs necessary to build and run Xamarin.Forms projects on Android by depending on xamarin-android.|
+| xamarin-android-llvm | The LLVM AOT compiler for Android, omitted by default from the xamarin-android component because it’s large and not all apps use it.<br><br>It may make sense for this to depend on the xamarin-android workload as it’s useless by itself.|
+
+# Versioning
+
+## Manifest Versioning
+
+Manifests follows a similar pattern to SDK versioning: each SDK band has a corresponding manifest band. Each SDK uses only the manifest band corresponding to its own band. Manifest versions however are completely independent of SDK patch levels; an SDK should use the latest available versions of the manifests for its band.
+
+To make this relationship more explicit, the SDK band for a manifest is encoded into the manifest’s ID e.g. `Microsoft.NET.Sdk.Android.SdkManifest-5.0.200`. The “version” of a manifest package can then be then simply a monotonic integer value.
+
+## Pack Versioning
+
+A manifest contains a list of pack IDs and their corresponding versions. A single version is defined for each pack: the most up-to-date version of that pack that is expected to be used with that SDK band.
+
+This intentionally separates pack versions from component definitions. A pack may be included in multiple workloads, and this separation ensures that all workloads get a consistent version of each pack. There’s also a semantic separation: the workload definition list describes which packs are included in each workload, while the pack version list describes the set of pack versions that are supported together.
+
+There are some cases where a single workload may want to contain side-by-side multiple versions of a single pack, or multiple components may want different versions of a single pack. This may apply, for example, to major/minor runtime versions, For these cases an aliasing mechanism is provided: a meaningful alias may be defined and mapped to a different version of a pack that is already included in the list. For example, `Foo-1.x` could be defined as an alias to version `1.0.5` of pack `Foo`, while `Foo` itself could be mapped to `2.1.4`. This is similar to how Linux libraries support side-by-side.
```

Comment:
It would be good to have an example to help motivate this additional complexity.

Reply:
Tried to explain this a bit better and added a link to the example.